### PR TITLE
GetMapping

### LINKED ISSUE #13: #backend
. GetMapping implemented and connection to mongoDB

### PR DESCRIPTION
**Pull request by Way-Maker7**
GetMapping implemented and connection to mongoDB

### CODE REVIEW

File: frontend/src/components/Gallery.tsx [hunk 1]
```diff
@@ -5,31 +5,46 @@ import GalleryItems from "./GalleryItems";
...
+            <input type="range" min="1" max="250" defaultValue={rangeValue}
+                   onChange={(e) => setRangeValue(e.target.value)}/>
+            <br/>
+            <br/>
+            {
+                radios.map((continents) => <div><input type="radio" id={continents} name="radioInput"
+                                                       defaultValue={radio}
+                                                       checked={radio === continents} onChange={(e) =>
+                    setRadio(e.target.id)}/>
+                    <label htmlFor={continents}>{continents}</label></div>)
```
Comment (JeTSkY1h) [inline frontend/src/components/Gallery.tsx:44]: Here you are giving some of the RadioInputs an ID with a space. e.g. "North Amerika" 
Iam not sure if this is working like intended here, but i'am certain that when u want to adress this element in your css you will have some trouble. 

You could solve this by using this selector: 
```css
input[id='North Amerika']{
 
}
```
But i would suggest to just delete the Spaces from the String. 
Also your formating seems off.
```javascript
radios.map(continent=>{
   return (
      <div>
         <input type=radio id={continents.replaceAll(" ","")} name="radioInput" defaultValue={radio} checked={radio===continents} onChange={e=>setRadio(e.target.id)}/>
         <label htmlFor={continents.replaceAll(" ","")}>{continent}</label>
      </div>
   )
})
```

File: backend/src/main/java/com/github/waymaker7/capstone/BlogsModel.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+@RestController
+@RequestMapping("/api/Blogs")
+
+public class BlogsModel {
+    @GetMapping
+    public List<Blogs> getTasks(){
+        return null;
+    }
+}
```
Comment (JeTSkY1h) [inline backend/src/main/java/com/github/waymaker7/capstone/BlogsModel.java:17]: what is this for? 
You already got a Controller. 
with exactly this methods in `BlogsController`.

**Review by JeTSkY1h — CHANGES_REQUESTED** [review]
You need to specify on which mongoDB instance your application should connect in your application.properties: 

```
spring.data.mongodb.uri= ${MONGODB_URI:mongodb://localhost:27017/<YOUR_DB_NAME>}
```
i think this is the error cause of which the build in your GirHub Action is failing. 

I will not Approve if your build is Failing.

File: backend/src/main/java/com/github/waymaker7/capstone/BlogsModel.java (hunk 2 above)
Comment (Way-Maker7) [reply backend/src/main/java/com/github/waymaker7/capstone/BlogsModel.java:17]: i have deleted it

**Comment by Way-Maker7** [comment]
Hi Bernd, die db sollte erstmal auf meinem Lokal laufen, später werde ich wie du geschrieben hast , konfigurieren

**Comment by JeTSkY1h** [comment]
Aber hier schlagen die Tests fehl. Ohne Konfiguration funktioniert die Datenbank auch Lokal nicht bei dir. 
Wie ich es geschrieben habe wäre die richtig Konfiguriert für deine Lokal über docker laufende MongoDB Instanz.

Wie gesagt, ich werde nicht approven solang die Checks rot sind.

**Pull request closed without merging**